.travis.yml

```diff
@@ -28,6 +28,16 @@ install:
   - git clone --depth=1 https://github.com/ksonnet/ksonnet-lib.git
   - export KUBECFG_JPATH=$PWD/ksonnet-lib
 
+before_script:
+  - |
+    # Hack to always build from github.com/kubeless/kubeless,
+    # to allow travis runs pushed from your own <user>/kubeless repo
```

Comment:
If you'd just clone the repo yourself ( ie in my case as `jjo/kubeless` ) and push it back
they build will deterministically fail to build as `github.com/jjo/kubeless` with interface
mismatches and alike (`jjo/kubeless` != `kubeless/kubeless`), from e.g.:
```
$ egrep -r github.com.*kubeless/kubeless pkg
pkg/utils/k8sutil.go: "github.com/kubeless/kubeless/pkg/spec"
pkg/controller/controller.go: "github.com/kubeless/kubeless/pkg/spec"
pkg/controller/controller.go: "github.com/kubeless/kubeless/pkg/utils"
```
I.e. it needs to be built from $GOPATH/src/github.com/kubeless/kubeless afaicf
(not a go expert, maybe there's a more elegant way ? ).